src/main/java/ee/aktors/misp2/configuration/DigiDoc4jConfiguration.java

```diff
@@ -54,6 +54,7 @@
     private static final String PARAM_TEST_MODE = "digidoc4j.test";
     private static final String PARAM_OCSP_SOURCE = "digidoc4j.ocsp";
     private static final String PARAM_TRUSTED_TERRITORIES = "digidoc4j.trustedTerritories";
+    private static final String PARAM_MID_ENABLED = "auth.mobileID";
     private static final String PARAM_MID_HOST = "mobileID.rest.hostUrl";
```

Comment:
The `mobileID.rest.hostUrl` isn't mentioned in the installation guide either. Is there a situation where its value should be adjusted by the MISP2 admin?